pkg/controller/machine_test.go

```diff
@@ -362,6 +387,108 @@ var _ = Describe("machine", func() {
 		)
 	})
 
+	Describe("#isMachineStatusEqual", func() {
+		type expect struct {
+			equal bool
+		}
+		type action struct {
+			s1 machinev1.MachineStatus
+			s2 machinev1.MachineStatus
+		}
+		type data struct {
+			action action
+			expect expect
+		}
+
+		lastOperation := machinev1.LastOperation{
+			Description:    "test operation",
+			LastUpdateTime: metav1.Now(),
+			State:          machinev1.MachineStateProcessing,
+			Type:           machinev1.MachineOperationCreate,
+		}
+		currentStatus := machinev1.CurrentStatus{
+			LastUpdateTime: lastOperation.LastUpdateTime,
+			Phase:          machinev1.MachinePending,
+			TimeoutActive:  true,
+		}
+
+		DescribeTable("##table",
+			func(data *data) {
+				stop := make(chan struct{})
+				defer close(stop)
+
+				c, trackers := createController(stop, testNamespace, nil, nil, nil)
+				defer trackers.Stop()
+
+				equal := c.isMachineStatusEqual(data.action.s1, data.action.s2)
```

Comment:
```suggestion
				equal := isMachineStatusEqual(data.action.s1, data.action.s2)
```
Tests are failing currently.

Reply:
Yes, the suggested changes seem to have broken both `check` and `test` step, can you please check.